module/VisualCeption.php

```diff
@@ -1,489 +1,498 @@
-<?php

```

Comment:
I use Visual Studio Code, automatic formatting was applied. In GitKraken set ignoring of white spaces, here is set and quitted.